Command/AtoumCommand.php

```diff
@@ -121,7 +121,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $reportCli->addWriter($writerCli);
         }
 
-        $runner->run($this->getAtoumArguments());
+        try {
+            $score = $runner->run()->getRunner()->getScore();
```

Comment:
I think you forgot to add the parameter `$this->getAtoumArguments()`to the run method because without it it's not working.
Without this parameters I have the message ``Error: Unable to add test file 'atoum'`.
But except that it's working nicely, good job man.